example-config.json

```diff
@@ -36,6 +36,7 @@
     "log_rotation_size": 2048,
     "log_directory_max_size": 51200,
     "log_rotation_hour_interval": 12,
+    "log_tag_style": "uint",
```

Comment:
All of this log stuff should be nested inside one `logging` object in my opinion, but im not sure best practices around changing the config file format.

Reply:
That's a very good point. I have no problem to change it if you think we can change the format for next version. I'm a bit hesitant to break backward compatibility but if it happens between releases then maybe that's ok. Especially if clio is not very wide spread at this moment. wdyt?

Reply:
I'm not really sure. I don't know how much operators read release notes. I really don't know best practices around changing a config file.